### ISSUE TITLE
 "Get started" button keyboard focus and background color contrast are 1.234:1 which is less than ratio 3:1: A11y_WCP URLs - ONNX Runtime_Home_Top Navigation_Color contrast.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20123]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11; #FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; #WCAG1.4.11; #Non-text Contrast; #A11yMAS;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".** 

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2

### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page is appeared.
3. Tab till "Top navigation".
4. Tab till "Get started" button.
5. Now Open Accessibility Insights Desktop app and open the color contrast analyzer.
6. Now check that "Get started" button keyboard focus and background color contrast are 1.234:1 which is less than ratio 3:1.

### Actual Results:
 "Get started" button keyboard focus and background color contrast are 1.234:1 which is less than ratio 3:1.
**Note:**
Similar issue observed with Blogs, docs, community, GitHub and themes buttons.

### Expected Results:
Color Contrast of "Get started" button focus should be meet the required ratio 3:1.

### User Impact:
Low visible user will face issue. There not able know where keyboard focus on the top navigation's elements.

### Attachment:

![Screenshot 2024-03-28 121840](https://github.com/microsoft/onnxruntime/assets/142479541/47a5c13d-4fdb-412a-aad7-e074bdb06e4c)

**Comment by v-jagansai** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.97 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
#Regressed:04-18-24; #Closed;

For reference attachment is added.
![Screenshot_19](https://github.com/microsoft/onnxruntime/assets/109065315/05026012-69da-41af-9f31-15d8d5c9074a)
